CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 platformdirs Changelog
 ======================
 
+platformdirs 3.0.0 (2023-02-05)
```

Comment:
Let's bump the date also by one

Reply:
did that.

but in case this takes longer, I suggest you just fix that yourself whenever you actually release it.